src/solver/ts-generator/availability.cpp

```diff
@@ -592,6 +609,23 @@ std::vector<Data::ThermalCluster*> getAllClustersToGen(const Data::AreaList& are
     return clusters;
 }
 
+listOfLinks getAllLinksToGen(Data::AreaList& areas)
+{
+    listOfLinks links;
+
+    areas.each(
+      [&links](const Data::Area& area)
+      {
+          std::ranges::for_each(area.links, [&links](auto& l)
+                                {
+                                    if (!l.second->tsGeneration.forceNoGeneration)
+                                        links.push_back(l.second);
+                                });
+      });
+
+    return links;
+}
+
 void writeResultsToDisk(const Data::Study& study,
```

Comment:
TimeSeries is a generic object, it doesn't specify if it's the one we generated or already existing ones
"results" is logic since we produce some data

Reply:
Fixed in #2200

Reply:
Now renamed into **writeTStoDisk**. Done in some further PR.